python/tk_multi_loader/dialog.py

```diff
@@ -1481,6 +1512,40 @@ def _populate_entity_breadcrumbs(self):
 
         self.ui.entity_breadcrumbs.setText("<big>%s</big>" % breadcrumbs)
 
+    def _status_filter_change(self):
+        """
+        Reload the status filter combo box 
+        With correct count in brackets
+        """
+        tmp_index = self._filter_status.currentIndex()
+        tmp_index = 0 if tmp_index< 0 else tmp_index
+
+        publish_items = self._publish_model._publish_items
+        self._filter_status.clear()
+
+        status_count = {"All": 0}
+        valid_ids = self._publish_proxy_model._valid_type_ids
+
+        for item in publish_items:
+            if item['type_id'] in valid_ids:
+                stat = item['sg_item']['sg_status_list']
+                if stat in status_count:
+                    status_count[stat] += 1
+                else:
+                    status_count[stat] = 1
+                status_count["All"] += 1
+
+        self._filter_status.addItem("All ({0})".format(status_count['All']))
+        for status in STATUS_LIST:
+            icon = QtGui.QIcon()
+            if status in status_count:
```

Comment:
You could even remove the whole status_count dict creation block by counting directly here
```python
count = sum(item['sg_item']['sg_status_list'] == status for item in valid_items))
```
with valid_items previously defined as 
```python
valid_items = [item for item in publish_items if item['type_id'] in valid_ids]
```